test/common/README.md

```diff
@@ -380,7 +380,8 @@ Synchronous version of `spawnPwd`.
 The `Countdown` module provides a simple countdown mechanism for tests that
 require a particular action to be taken after a given number of completed
 tasks (for instance, shutting down an HTTP server after a specific number of
-requests).
+requests). Note that _the Countdown will fail the test if the remainder
+did not reach 0_.
```

Comment:
The italics aren't necessary here, IMO.

Reply:
I would go so far and remove the `Note that` as well.